File: 1391-CountingElements/1391-CountingElements.java
// Last updated: 10/8/2025, 9:49:12 PM
class Solution {
    public int countElements(int[] arr) {
        Set<Integer> hs=new HashSet<>();
        for(int x :arr){
            hs.add(x);
        }
        int count=0;
        for(int x:arr){
            if(hs.contains(x+1)){
                count++;
            }
        }
        return count;
    }
}

File: 1391-CountingElements/1391-CountingElements.py
# Last updated: 10/8/2025, 9:49:10 PM
class Solution:
    def countElements(self, arr: List[int]) -> int:
        seen=set(arr)
        count=0
        for i in range(len(arr)):
            if arr[i]+1 in seen:
                count+=1
        return count
        